File: src/main/java/com/etour/app/dto/PassengerResponseDTO.java
package com.etour.app.dto;

import java.math.BigDecimal;

public class PassengerResponseDTO {

    private String passengerName;
    private String passengerType;
    private BigDecimal passengerAmount;

    public String getPassengerName() {
        return passengerName;
    }

    public void setPassengerName(String passengerName) {
        this.passengerName = passengerName;
    }

    public String getPassengerType() {
        return passengerType;
    }

    public void setPassengerType(String passengerType) {
        this.passengerType = passengerType;
    }

    public BigDecimal getPassengerAmount() {
        return passengerAmount;
    }

    public void setPassengerAmount(BigDecimal passengerAmount) {
        this.passengerAmount = passengerAmount;
    }
}

File: dotnet/backend/DTO/Booking/PassengerResponseDTO.cs
namespace Etour_Backend_dotnet.DTO.Booking;

public class PassengerResponseDTO
{
    public int PassengerId { get; set; }
    public string PassengerName { get; set; } = null!;
    public DateOnly DateOfBirth { get; set; }
    public string Gender { get; set; } = null!;
    public string PassengerType { get; set; } = null!;
    public decimal PassengerAmount { get; set; }
}
